t/plugin/limit-count-redis-cluster.t

```diff
@@ -95,8 +95,7 @@ __DATA__
                             "policy": "redis-cluster",
                             "redis_timeout": 1001,
                             "redis_cluster_nodes": [
-                                "127.0.0.1:5000",
-                                "127.0.0.1:5001"
+                                "127.0.0.1:5000"
```

Comment:
please add a new test case instead of change old one.
Better cover example in https://github.com/apache/apisix/issues/10596#issuecomment-1844685990, like
```
"limit-count": {
            "redis_cluster_nodes": [
              "redis-headless-service:6379"
            ],
        }
```